File: Python/백준/Bronze/21866. 추첨을 통해 커피를 받자/추첨을 통해 커피를 받자.py
aa = [100, 100, 200, 200, 300, 300, 400, 400, 500]
bb = list(map(int,input().split()))
h = 0
for i in range(9):
    if aa[i]<bb[i]:
        h=1
        break
if h==1:
    print('hacker')
elif sum(bb)<100:
    print('none')
else:
    print('draw')

File: C#/백준/Bronze/21866. 추첨을 통해 커피를 받자/추첨을 통해 커피를 받자.cs
using System;
using System.Linq;
namespace q{
    class w{
        static void Main(String[] args){
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int[] cc = {100, 100, 200, 200, 300, 300, 400, 400, 500};
            int h = 0;
            for (int i=0;i<9;i++){
                if (aa[i]>cc[i]){
                    h=1;
                    break;
                }
            }
            if (h==1){
                Console.WriteLine("hacker");
            }
            else if (aa.Sum()>=100){
                Console.WriteLine("draw");
            }
            else{
                Console.WriteLine("none");
            }
        }
    }
}